pyqtgraph/graphicsItems/BarGraphItem.py

```diff
@@ -58,6 +58,16 @@ def setOpts(self, **opts):
         self.update()
         self.informViewBoundsChanged()
         
+    def setPen(self, *args, **kargs):
+        """Set the pen used to draw the curve."""
+        self.setOpts(pen = fn.mkPen(*args, **kargs))
+
+
```

Comment:
Nitpick: can remove this blank like so there's only one between methods. Also after `setBrush`